client/blocks/video-editor/index.jsx

```diff
@@ -0,0 +1,230 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { noop } from 'lodash';
+import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import ProgressBar from 'components/progress-bar';
+import Notice from 'components/notice';
+import DetailPreviewVideo from 'post-editor/media-modal/detail/detail-preview-video';
+import VideoEditorButtons from './video-editor-buttons';
+import { updatePoster } from 'state/ui/editor/video-editor/actions';
+import {
+	getPosterUploadProgress,
+	getPosterUrl,
+	shouldCloseVideoEditorModal,
+	shouldShowVideoEditorError,
+} from 'state/selectors';
+
+/**
+ * Module variables
+ */
+let isSelectingFrame = false;
```

Comment:
Yeah, that's strange. Fixed in https://github.com/Automattic/wp-calypso/pull/11648/commits/3e417f95f316a8ce9c4353ea744cd2f1af99b194.